tests/unit/s2n_signature_algorithms_test.c

```diff
@@ -861,41 +1221,36 @@ int main(int argc, char **argv)
 
         config->security_policy = &test_security_policy;
 
-        /* Test: no shared valid signature schemes, using TLS1.3. Server cant pick preferred */
+        /* Test: no shared valid signature schemes, using TLS1.3. Server can't pick preferred */
```

Comment:
Are these two tests covered by your new "Test: no rsa certs" test? Or are these adding something else?

Reply:
Technically? But what they're really trying to test is fallback logic specifically with PSS, which is tricky to cover in tests that can't make assumptions about the libcrypto.

I cleaned these tests up and added them to the preceding section that's specifically testing PSS behavior.